### ISSUE TITLE
Wrongly triggers error for comment in styled-component context

### ISSUE THREAD

**Issue by emilie-t-ledn** [stylelint/postcss-css-in-js#243]
### What steps are needed to reproduce the bug?

Node `v14.18.1`

Packages versions (removed irrelevant ones):

```
 {
    "babel-plugin-styled-components": "1.13.2",
    "postcss": "8.4.4",
    "postcss-scss": "4.0.2",
    "postcss-styled": "0.34.0",
    "postcss-syntax": "0.36.2",
    "prettier": "2.3.1",
    "react": "17.0.2",
    "react-dom": "17.0.2",
    "styled-components": "5.3.0",
    "stylelint": "14.1.0",
    "stylelint-config-recommended": "6.0.0",
    "typescript": "4.3.2",
},
```

### What did you expect to happen?

Comment should not trigger any error

### What actually happened?

`Unknown word  CssSyntaxError`.

It shows the first comment  (`// JS type comment`) inside a styled definition (styled.button``) as error.
There is a comment below the first comment that is not shown as an error.
If I remove the first comment, the second comment becomes an error.
Changing to CSS type comment removes the error.

```
export const IconButton = styled.button`
  // First coment
  ${({ buttonType }) =>
    buttonType !== 'something' &&
    css`
      ${BaseButtonStyle};
    `}

  // Second comment
  ${({ buttonType }) =>
    buttonType === 'something' &&
    css`
      line-height: 0;
      padding: 0;
    `}
`;
```
